projectile.el

```diff
@@ -3182,25 +3182,38 @@ Should be set via .dir-locals.el.")
 It takes precedence over the default command for the project type when set.
 Should be set via .dir-locals.el.")
 
+(defun projectile-default-generic-command (project-type command-type)
+  "Generic retrieval of COMMAND-TYPEs default cmd-value for PROJECT-TYPE.
+
+If found, checks if value is symbol or string. In case of symbol resolves
+to function `funcall's. Return value of function MUST be string to be executed as command."
+  (let ((plist-retrieved-generic-command (plist-get (gethash project-type projectile-project-types) command-type)))
+    (cond
+     ((stringp plist-retrieved-generic-command) plist-retrieved-generic-command)
+     ((symbolp 'plist-retrieved-generic-command)
```

Comment:
This doesn't look correct to me - here you're passing a quoted symbol to symbolp, which means this will always eval to true.